app/integrators/covid_api_v2_integrator.py

```diff
@@ -24,8 +24,7 @@
                                          TimeseriesUSDataModel,
                                          TimeseriesUSInfoModel,
                                          TimeseriesUSModel, TotalModel)
-from utils.get_data import (get_data_daily_reports,
-                              get_data_daily_reports_us, get_data_lookup_table,
+from utils.get_data import (DailyReports, get_data_lookup_table,
```

Comment:
**get_country** also needs to change to

`self.df = self.daily_reports.get_data_daily_reports() # Get base data`